flow.yml

```diff
@@ -24,7 +24,7 @@ executors:
     uses: GLID3Diffusion
     uses_with:
       glid3_path: ../glid-3-xl
-      clip_as_service_url: https://demo-cas.jina.ai:8443
+      clip_blank_encoding_file_path: ./clip_blank_encoding.json
```

Comment:
If we get it from file, I would rather not add it as config. This will be unique and I think is better to have it hardcoded in the `Executor` image